bendowen/BEN_DOWEN.md

```diff
@@ -0,0 +1,74 @@
+# Tester of The Day
+
+## Origin story
+
+The story starts, as with all good things, with a joke. Dan Ashby had started a rant on Twitter about Paid-for awards, around the end of July 2020.
+Lot's of people joined in and thread and the jokes rumbled on for a few days. Many of us making more outranges sounding awards.
+
+You can checkout the thread for yourself: <https://twitter.com/DanAshby04/status/1288763581333413888>
+
+Inspired by the humour of this all, I thought, what if? What if I made a daily award, a free award?
+
+At first I thought I would simply find all the winners myself. Simply look on Twitter, find someone saying something cool about Software Testing and then boom, award them Tester of The Day.
+
+A few days in, and I was thinking maybe this will last a week. But then something amazing happened. The awesome power of the Software Testing Community awoke. People started to nominate people to win Tester of the Day.
+
+The community were liking and following the Winners, congratulating them, celebrating them. Some of the early supporters like Chris Armstrong, Arleen Andrews and Marie Drake went on to nominate multiple winners.
+
+Momentum started to build, and I felt an obligation, and some considerable excitement, to keep going. So I built a site, while on a Summer Holiday at my Mum's Seaside Chalet, a precious thing in 2020.
+
+## Celebrating the Community
+
+At the time of writing, more Tester of The Day has been awarded more then 150 times since it was started on the 17th of August 2020. With only a handful of people winning more then once.
+
+I am very humbled by the joy and excitement many winners have expressed. Here are just a few glorious examples:
+
```

Comment:
Feels like these 3 personal accounts could be under the a same subheading. e.g. `### Winners Tesimonials`